File: template/node/function/handler.ts
"use strict";

import { IMessage } from "qq-guild-bot";
import { OpenAPI } from "../pkg/openapi";

// ts还没支持,先写在这里了

/**
 * const bot = require('qq-guild-bot');
 * @var {*} api bot.OpenAPI
 */
let api = OpenAPI();

export function handle(msg: IMessage) {
  api.messageApi.postMessage(msg.channel_id, {
    content: `<@!` + msg.author.id + `>ok`,
    msg_id: msg.id,
  });
}

File: template/go/function/handler.go
package function

import (
	"context"
	"handler/pkg/openapi"

	"github.com/tencent-connect/botgo/dto"
)

var api = openapi.OpenAPI()

// Handle a qqbot request
func Handle(msg *dto.WSATMessageData) (*dto.Message, error) {
	return api.PostMessage(context.Background(), msg.ChannelID, &dto.MessageToCreate{
		Content:          "<@!" + msg.Author.ID + ">ok",
		MsgID:            msg.ID,
		MessageReference: nil,
		Markdown:         nil,
	})
}
